<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFileOperatorDlg</class>
 <widget class="QDialog" name="CFileOperatorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1691</width>
    <height>910</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1691</width>
    <height>910</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1691</width>
    <height>910</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CFileOperatorDlg{
background-color:rgb(255,254,255);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>560</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
background-color:rgb(236,239,246);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
background-color:rgb(254,254,254);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TestTestTest(8945247)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3
{
color:rgb(83,86,93);
}</string>
    </property>
    <property name="text">
     <string>本地设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/back.png</normaloff>:/back.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
background-color:rgb(221,58,95);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/up.png</normaloff>:/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/home.png</normaloff>:/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/refalsh.png</normaloff>:/refalsh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>431</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit
{
background-color:rgb(255,253,255);
border:none;
}</string>
    </property>
    <property name="placeholderText">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5
{
color:rgb(206,234,254);
}</string>
    </property>
    <property name="text">
     <string>发送   &gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>811</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  color:#666666;
  padding: 1px 15px 1px 3px;
  border: rgba(228,228,228,1);
}

QComboBox::drop-down
{
border-image:url(:/comboxDowm.png);
min-width:30px;
min-heigth:30px;
}

QComboBox#comboBox::drop-down:checked
{
border-image:url(:/comboxUp.png);
}

QComboBox QAbstractItemView{
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	font-size:14px;
    outline: 0px;  
 }




</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>811</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 设置QTableView表头的背景颜色和文字颜色 */  
QTableView QHeaderView::section {  
    background-color:rgb(236,239,246); /* 背景色 */  
    color: rgb(155,155,155);         /* 文字颜色 */
    height:40px;  
    padding-left: 4px;    /* 左内边距 */  
    padding-right: 4px;   /* 右内边距 */  
    border-left: 1px solid #555; /* 左边框 */  
    border-right: 1px solid #555; /* 右边框 */  
}  
  
/* 当鼠标悬停在表头部分时 */  
QTableView QHeaderView::section:hover {  
  /*  background-color: #555; /* 鼠标悬停时的背景色 */  
}  
  
/* 表头被选中的样式 */  
QTableView QHeaderView::section:selected {  
    background-color: #a6cafe; /* 选中时的背景色 */  
    color: black;             /* 选中时的文字颜色 */  
}  
  
/* 当表头不可用时（比如某个方向被禁用） */  
QTableView QHeaderView::section:disabled {  
    color: gray; /* 禁用时的文字颜色 */  
}  
  
/* 垂直表头的样式 */  
QTableView QHeaderView::section:vertical {  
    border-top: 1px solid #555; /* 上边框 */  
    border-bottom: 1px solid #555; /* 下边框 */   
}  
  
/* 表头的分隔线样式（注意：这个可能需要结合具体的Qt版本和平台来看是否支持） */  
QTableView QHeaderView::separator {  
    background-color: rgb(0,0,0); /* 分隔线颜色 */  
    width: 3px;              /* 分隔线宽度 */  
    height: 3px;             /* 垂直分隔线的高度（对于水平分隔线可能不适用） */  
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_11{
background-color:rgb(255,253,255);
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/search.png</normaloff>:/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>820</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{
background-color:rgb(236,239,246);
}</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{
background-color:rgb(254,254,254);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TestTestTest(8945247)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_7
{
color:rgb(83,86,93);
}</string>
    </property>
    <property name="text">
     <string>远程设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/back.png</normaloff>:/back.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/up.png</normaloff>:/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/home.png</normaloff>:/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/refalsh.png</normaloff>:/refalsh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_8
{
background-color:rgb(255,253,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/search.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>431</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_2
{
background-color:rgb(255,253,255);
border:none;
}</string>
    </property>
    <property name="placeholderText">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_10
{
color:rgb(206,234,254);
}</string>
    </property>
    <property name="text">
     <string>发送   &gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>811</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  color:#666666;
  padding: 1px 15px 1px 3px;
  border: rgba(228,228,228,1);
}

QComboBox::drop-down
{
border-image:url(:/comboxDowm.png);
min-width:30px;
min-heigth:30px;
}

QComboBox#comboBox::drop-down:checked
{
border-image:url(:/comboxUp.png);
}

QComboBox QAbstractItemView{
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	font-size:14px;
    outline: 0px;  
 }
</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>811</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 设置QTableView表头的背景颜色和文字颜色 */  
QTableView QHeaderView::section {  
    background-color:rgb(236,239,246); /* 背景色 */  
    color: rgb(155,155,155);         /* 文字颜色 */
    height:40px;  
    padding-left: 4px;    /* 左内边距 */  
    padding-right: 4px;   /* 右内边距 */  
    border-left: 1px solid #555; /* 左边框 */  
    border-right: 1px solid #555; /* 右边框 */  
}  
  
/* 当鼠标悬停在表头部分时 */  
QTableView QHeaderView::section:hover {  
  /*  background-color: #555; /* 鼠标悬停时的背景色 */  
}  
  
/* 表头被选中的样式 */  
QTableView QHeaderView::section:selected {  
    background-color: #a6cafe; /* 选中时的背景色 */  
    color: black;             /* 选中时的文字颜色 */  
}  
  
/* 当表头不可用时（比如某个方向被禁用） */  
QTableView QHeaderView::section:disabled {  
    color: gray; /* 禁用时的文字颜色 */  
}  
  
/* 垂直表头的样式 */  
QTableView QHeaderView::section:vertical {  
    border-top: 1px solid #555; /* 上边框 */  
    border-bottom: 1px solid #555; /* 下边框 */   
}  
  
/* 表头的分隔线样式（注意：这个可能需要结合具体的Qt版本和平台来看是否支持） */  
QTableView QHeaderView::separator {  
    background-color: rgb(0,0,0); /* 分隔线颜色 */  
    width: 3px;              /* 分隔线宽度 */  
    height: 3px;             /* 垂直分隔线的高度（对于水平分隔线可能不适用） */  
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1691</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget {  
}  
  
QTabBar {  
    /* 设置标签栏的整体样式 */  
    background-color: #f0f0f0; /* 背景颜色 */  
    border: 1px solid #d3d3d3; /* 边框 */  
}  
  
QTabBar::tab {  
    /* 设置每个标签的样式 */  
    /*background-color: #e0e0e0;*/ /* 背景颜色 */ 
    background-color:rgba(0,0,0,0); 
    color: black; /* 文字颜色 */  
    padding: 5px; /* 内边距 */  
    /*border: 1px solid transparent;*/ /* 边框，初始设置为透明，以便在选中时更改 */  
    border: none;
    border-bottom-right-radius: 4px; /* 右下圆角（如果需要） */  
    border-top-right-radius: 4px; /* 右上圆角（如果需要） */  
    min-width: 80px; /* 最小宽度 */  
}  
  
QTabBar::tab:selected {  
    /* 设置选中标签的样式 */  
    background-color: #3399ff; /* 选中时的背景颜色 */  
    color: white; /* 选中时的文字颜色 */  
    border-color: #3399ff; /* 选中时的边框颜色 */  
}  
  
QTabBar::tab:hover {  
    /* 设置鼠标悬停在标签上的样式 */  
    background-color: #99ccff; /* 悬停时的背景颜色 */  
}  </string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>传输列表</string>
    </attribute>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1690</width>
       <height>293</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_3
{
background-color: rgb(236,239,246); /* 背景颜色 */  
}</string>
     </property>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1690</width>
        <height>293</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 设置QTableView表头的背景颜色和文字颜色 */  
QTableView QHeaderView::section {  
    background-color:rgb(236,239,246); /* 背景色 */  
    color: rgb(155,155,155);         /* 文字颜色 */
    height:40px;  
    padding-left: 4px;    /* 左内边距 */  
    padding-right: 4px;   /* 右内边距 */  
    border-left: 1px solid #555; /* 左边框 */  
    border-right: 1px solid #555; /* 右边框 */  
}  
  
/* 当鼠标悬停在表头部分时 */  
QTableView QHeaderView::section:hover {  
  /*  background-color: #555; /* 鼠标悬停时的背景色 */  
}  
  
/* 表头被选中的样式 */  
QTableView QHeaderView::section:selected {  
    background-color: #a6cafe; /* 选中时的背景色 */  
    color: black;             /* 选中时的文字颜色 */  
}  
  
/* 当表头不可用时（比如某个方向被禁用） */  
QTableView QHeaderView::section:disabled {  
    color: gray; /* 禁用时的文字颜色 */  
}  
  
/* 垂直表头的样式 */  
QTableView QHeaderView::section:vertical {  
    border-top: 1px solid #555; /* 上边框 */  
    border-bottom: 1px solid #555; /* 下边框 */   
}  
  
/* 表头的分隔线样式（注意：这个可能需要结合具体的Qt版本和平台来看是否支持） */  
QTableView QHeaderView::separator {  
    background-color: rgb(0,0,0); /* 分隔线颜色 */  
    width: 3px;              /* 分隔线宽度 */  
    height: 3px;             /* 垂直分隔线的高度（对于水平分隔线可能不适用） */  
}</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>传输日志</string>
    </attribute>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1690</width>
       <height>293</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_4
{
background-color: rgb(236,239,246); /* 背景颜色 */  
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
